src/gui/torrentcreatordialog.cpp

```diff
@@ -55,6 +55,7 @@ TorrentCreatorDialog::TorrentCreatorDialog(QWidget *parent, const QString &defau
     , m_storeStartSeeding(SETTINGS_KEY("StartSeeding"))
     , m_storeIgnoreRatio(SETTINGS_KEY("IgnoreRatio"))
     , m_storeOptimizeAlignment(SETTINGS_KEY("OptimizeAlignment"), true)
+    , m_padFileLimit(SETTINGS_KEY("PadFileLimit"), -1)
```

Comment:
>Align to piece boundary for files larger than

According to the option description "PadFileLimit" isn't correct name (at least it's not obvious enough). It should be something like "UnalignedFileSizeLimit".

Reply:
>It should be something like "UnalignedFileSizeLimit".

I don't think it is easy to correlate unaligned term to our use case. How about "PieceBoundaryFileSizeLimit"?

Reply:
>`PadFileSizeLimit` sounds easier both to read and to understand (at least to me).

Are we talking about the size of some "pad file"? IIRC, it's about some file transferred by torrent that can be either aligned (i.e. padded) or not. Myabe `PaddedFileSizeLimit`?

Reply:
>Myabe PaddedFileSizeLimit?

OK, updated.